catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/Dod520001Attributes.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.catalog.core.api.impl.types;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import java.util.HashSet;
+import java.util.Set;
+import org.codice.alliance.catalog.core.api.types.Dod520001;
+
+/**
+ * <b> This code is experimental. While this is functional and tested, it may change or be removed
+ * in a future version of the library. </b>
+ */
+public class Dod520001Attributes implements Dod520001, MetacardType {
+  private static final Set<AttributeDescriptor> DESCRIPTORS = new HashSet<>();
+
+  private static final String NAME = "dod520001";
+
+  static {
+    DESCRIPTORS.add(
+        new AttributeDescriptorImpl(
+            Dod520001.SECURITY_DOD5200_AEA,
+            true /* indexed */,
```

Comment:
❓ Did you mean to change all of these to be indexed? None of them were, previously.

Reply:
I did change that. Should they not be? All our other security attributes are so I figured these should be as well.

Reply:
That's probably a good change then. I just wanted to confirm.